bazel/foreign_cc/BUILD

```diff
@@ -20,12 +20,14 @@ configure_make(
     }),
     lib_source = "@com_github_gperftools_gperftools//:all",
     linkopts = ["-lpthread"],
-    make_commands = ["make install-libLTLIBRARIES install-perftoolsincludeHEADERS"],
-    static_libraries = select({
+    out_static_libs = select({
         "//bazel:debug_tcmalloc": ["libtcmalloc_debug.a"],
         "//conditions:default": ["libtcmalloc_and_profiler.a"],
     }),
     tags = ["skip_on_windows"],
+    targets = [
+        "install-libLTLIBRARIES install-perftoolsincludeHEADERS",
```

Comment:
Doing this in 1 string uses 1 make command like before, I'm not sure if it matters but I kept it the same